# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cdrkit/hotfix-cmake-policy.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cdrkit-1.1.11/CMakeLists.txt.vanilla	2025-08-25 10:37:12.179852720 +0200
+++ cdrkit-1.1.11/CMakeLists.txt	2025-08-25 10:37:14.697591312 +0200
@@ -1,3 +1,4 @@
+cmake_minimum_required(VERSION 3.5)
 PROJECT (cdrkit C)
 SUBDIRS(include genisoimage wodim libedc libhfs_iso libparanoia icedax libusal librols libunls readom netscsid 3rd-party/dirsplit)
 
